sympy/matrices/dense.py

```diff
@@ -1281,16 +1281,16 @@ def hessian(f, varlist, constraints=[]):
     return out
 
 
-def jordan_cell(eigenval, n):
+def jordan_block(eigenval, n):
```

Comment:
@siefkenj Do you mean we have to use the same kind of deprecated function that we used in other files ?